openqabot/loader/qem.py

```diff
@@ -124,22 +127,21 @@ def get_incident_settings_data(token: dict[str, str], number: int) -> Sequence[D
 def get_incident_results(inc: int, token: dict[str, str]) -> list[dict[str, Any]]:
     settings = get_incident_settings(inc, token, all_incidents=False)
 
-    ret = []
+    all_data = []
     for job_aggr in settings:
         data = get_json("api/jobs/incident/" + f"{job_aggr.id}", headers=token)
-        ret += data
         if "error" in data:
             raise ValueError(data["error"])
-
-    return ret
+        all_data.append(data)
+    return list(chain.from_iterable(all_data))
```

Comment:
`chain.from_iterable` does flatten `all_data` which was previously not done. Why is this change needed? Or do I understand `from_iterable` wrong?